Microsoft.DotNet.Interactive/Events/DisplayedValue.cs

```diff
@@ -2,18 +2,17 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
-using System.Threading.Tasks;
-using Microsoft.DotNet.Interactive.Commands;
 using Microsoft.DotNet.Interactive.Formatting;
 
 namespace Microsoft.DotNet.Interactive.Events
 {
-    public class DisplayedValue 
+    public class DisplayedValue
     {
         private readonly string _displayId;
         private readonly string _mimeType;
+        private readonly KernelInvocationContext _context;
 
-        public DisplayedValue(string displayId, string mimeType)
+        public DisplayedValue(string displayId, string mimeType, KernelInvocationContext context)
```

Comment:
how will this impact the serialization of events?

Reply:
This type is not serialized.